backend/src/main/java/com/bakdata/conquery/apiv1/query/SecondaryIdQuery.java

```diff
@@ -127,23 +126,6 @@ public void resolve(final QueryResolveContext context) {
 		}
 	}
 
-	/**
-	 * selects the right column for the given secondaryId from a table
-	 */
-	@CheckForNull
-	private Column findSecondaryIdColumn(Table table) {
```

Comment:
Den Teil hier habe ich in die `Table` Klasse verschoben und public gemacht, damit ich bei der Conversion auch darauf zugreifen kann.